src/ReleaseHistory.md

```diff
@@ -13,11 +13,13 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
-## **v4.0.0** UNRELEASED
+## **v4.1.0** UNRELEASED
+
+## **v4.0.0**
 * BUG: Fix unhandled `ArgumentException` in `Enum.TryParse` on passing `PdbLoad` value to `--trace` command-line argument. [821](https://github.com/microsoft/binskim/pull/821)
 
 ## **v4.0.0-rc4**
-* DEP: Update Sarif.Sdk submodule from [235394a to 120fae35](https://github.com/microsoft/sarif-sdk/compare/235394a...120fae35). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/120fae35/src/ReleaseHistory.md).
```

Comment:
>120fae35 [](http://example.com/codeflow?start=51&length=8)

I used 8 instead of 7 which was by mistake. It works with either 7 or 8.
a quick try the min working here is 5 :)